src/wast-parser.cc

```diff
@@ -1324,8 +1293,53 @@ bool WastParser::PeekIsCustom() {
          tokens_.front().text() == "custom";
 }
 
+Result WastParser::ResolveRefTypes(const Module* module,
+                                   TypeVector& types,
+                                   ReferenceVars& ref_vars,
```

Comment:
I think the style we use it either a const ref (for const arguments) or pointer to mutable argument.    So these two args should either be const ref of pointer I think.